modules/installation-uninstall-clouds.adoc

```diff
@@ -119,7 +125,18 @@ $ ccoctl ibmcloud delete-service-id \
     --name <cluster_name>
 ----
 endif::ibm-cloud[]
-// The above CCO credential removal for IBM Cloud is only necessary for manual mode. Future releases that support other credential methods will not require this step.
+
+ifdef::nutanix[]
+. Remove the manual CCO credentials that were created for the cluster:
++
+[source,terminal]
+----
+$ ccoctl nutanix delete-service-id \
+    --credentials-requests-dir <path_to_credential_requests_directory> \
+    --name <cluster_name>
+----
+endif::nutanix[]
+// The above CCO credential removal for IBM Cloud and Nutanix is only necessary for manual mode. Future releases that support other credential methods will not require this step.
```

Comment:
This section should not be required as the ccoutl is not creating credentials in the nutanix environment.

Reply:
Indeed, `delete-service-id` is not a valid subcommand for `ccoctl nutanix`.

Reply:
Done.